_docs-2/configuration/overview.md

```diff
@@ -86,13 +86,26 @@ client.tableOperations().setProperty("mytable", "table.durability", "log");
 
 ### Zookeeper Considerations
 
+#### Storing Accumulo Server Properties
+
```

Comment:
This entire page is content describing how Accumulo configuration works. I don't think ZooKeeper configuration tips should be put here, as it is out of scope for this page.

Also, I don't think it's necessary to modify this page at all (see other review comments).

```suggestion
```

Reply:
I think your points are valid - the issue that I had in creating this was were would I go, if on an upgrade, the monitor stopped showing the ZooKeeper info.  The configuration section seemed like a place I might go first - the monitor pages seems to provide more general info that existing users may skip - but, it is a monitor issue, so, maybe there?  And rather than duplicate info, the link.

The essence I what I wanted to convey is that the monitor relies on the stat command.  Other ZooKeeper operations are using a connector and will continue to work - but the monitor is using an alternate connection path and that may not be obvious users.  With the zookeeper white list changes, they need to do update their existing  zookeeper configuration. 

Fixing uno helps new instance experiences - what I hoped to convey was that exiting users may need adjustments - and those might not have been obvious

Reply:
The main intent I have in my feedback is to keep the documentation concise. We already have so much documentation, it's overwhelming and critical information gets buried in pages and pages of superfluous explanatory text. Every addition to the docs that focuses on one doc deficiency at a time risks making the situation slightly worse, which builds up over time. I just want to avoid that and try to think about the impact on the totality of the documentation by keeping the additions concise.

> The essence I what I wanted to convey is that the monitor relies on the stat command.

Right. I think that's the best thing to focus on. I just want it to be concise.

As for where this should live, I think your original "Note" is fine (did my proposed changes to that satisfy your intent in a concise way?). It could also live in the "Troubleshooting" section.